lib/jwt/__main__.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
```

Comment:
Better to include the libs, using a separate PR per lib. Then after, rebase this PR on that.